sophiread/CMakeLists.txt

```diff
@@ -31,17 +31,17 @@ file(COPY ${CMAKE_SOURCE_DIR}/resources/data DESTINATION ${CMAKE_BINARY_DIR})
 add_compile_options(
   -O3
   -std=c++20
-  -march=native
+#  -march=native
   -ffast-math
   -pthread
   -Wall
   -Wextra
-  # ${OpenMP_CXX_FLAGS}
+  ${OpenMP_CXX_FLAGS}
 )
 
 # Add Sophiread library
 # NOTE: this will take care of the testing as well
-# add_subdirectory(SophireadLib)
+add_subdirectory(SophireadLib)
```

Comment:
The performance library is `FastSophiread`, both `SophireadLib` and `SophireadCLI` are intended for slow offline processing.